### PR TITLE
Fix find multiple elements

### PR DESCRIPTION
**Pull request by NA070**

### CODE REVIEW

File: lib/bucky/test_equipment/pageobject/base_pageobject.rb [hunk 1]
```diff
@@ -55,7 +55,8 @@ def find_elem(method, value)
 
           elem = @driver.find_elements(method_name, value)
           raise_if_element_empty(elem, method_name, value)
-          elem
+          Selenium::WebDriver::Element.class_eval { define_method('[]') { |num| elem[num] } }
```
Comment (rueyjye) [inline lib/bucky/test_equipment/pageobject/base_pageobject.rb]: I don't think it's a good idea using define_method in every find_elem.
Maybe adding like this at the first of BasePageObject class.
```
Selenium::WebDriver::Element.class_eval { define_method('[]') { |num| self[num] } }
```

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml [hunk 2]
```diff
@@ -80,4 +80,52 @@ cases:
           page: index
           part:
             locate: links
-            num: 1
\ No newline at end of file
+            num: 1
+  - case_name: pc_e2e_6
+    func: pc e2e 6 get element from PageObject
+    desc: pc e2e 6 get element from PageObject
```
Comment (rueyjye) [inline system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Maybe 「pc e2e 6 click the single element.」is better.

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml [hunk 3]
```diff
@@ -80,4 +80,52 @@ cases:
...
+        exec:
+          operate: go
+          url: http://bucky.net
+      - proc: index
+        exec:
+          operate: click_single_element
+          page: index
+  - case_name: pc_e2e_7
+    func: pc e2e 7 get multiple element from PageObject
+    desc: pc e2e 7 get multiple element from PageObject
```
Comment (rueyjye) [inline system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Maybe 「pc e2e 7 click the second element in multiple element .」is better.

File: system_testing/test_bucky_project/services/service_a/pc/verifications/index.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'bucky/test_equipment/verifications/e2e_verification'
+module Services
+  module ServiceA
+    module Pc
+      module Verifications
+        class Index < Bucky::TestEquipment::Verifications::E2eVerification
+          def click_single_element(_)
+            @pages.index.links.click
```
Comment (rueyjye) [inline system_testing/test_bucky_project/services/service_a/pc/verifications/index.rb]: I think it is available to operate the element in verification.
But In specification, can we make the verify method without using assert?

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml [hunk 5]
```diff
@@ -80,4 +80,52 @@ cases:
...
+        exec:
+          operate: go
+          url: http://bucky.net
+      - proc: index
+        exec:
+          verify: click_single_element
+          page: index
+  - case_name: pc_e2e_9
+    func: pc e2e 9 get multiple element from Verifications
+    desc: pc e2e 9 get multiple element from Verifications
```
Comment (rueyjye) [inline system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Maybe 「pc e2e 9 click the second element in multiple element from Verifications.」is better.

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml [hunk 6]
```diff
@@ -80,4 +80,52 @@ cases:
...
+        exec:
+          operate: go
+          url: http://bucky.net
+      - proc: index
+        exec:
+          operate: click_multiple_element
+          page: index
+  - case_name: pc_e2e_8
+    func: pc e2e 8 get element from Verifications
+    desc: pc e2e 8 get element from Verifications
```
Comment (rueyjye) [inline system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Maybe 「pc e2e 8 click the single element from Verifications」is better.

File: lib/bucky/test_equipment/pageobject/base_pageobject.rb (hunk 1 above)
Comment (NA070) [reply lib/bucky/test_equipment/pageobject/base_pageobject.rb]: Fixed!

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml (hunk 2 above)
Comment (NA070) [reply system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Fixed!

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml (hunk 3 above)
Comment (NA070) [reply system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Fixed!

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml (hunk 6 above)
Comment (NA070) [reply system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Fixed!

File: system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml (hunk 5 above)
Comment (NA070) [reply system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml]: Fixed!

**Pull request merged**